extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -195,4 +206,37 @@ export class DeployConfigPage extends DacFxConfigPage {
 		this.databaseLoader.loading = false;
 		return true;
 	}
+	/*
+	Function that is used to change the radio button DOM based on whether databases exist or not.
+	*/
+	private async disableUpgradeRadioButton(upgradeRadioButton: azdata.RadioButtonComponent, newRadioButton: azdata.RadioButtonComponent): Promise<azdata.FormComponent> {
+		// Set the upgrade radio button to be disabled as well and set the default options for the
+		// new databases radio button as defined in the createRadiobuttons function.
+		upgradeRadioButton.enabled = false;
```

Comment:
instead of having this code be pretty much the same thing that's in newRadioButton.onDidClick(), can you refactor this to move the newRadioButton click content to a helper function that can be called in these two places?

Reply:
Acknowledged, I will create helper functions for the upgrade radio button, new radio button and for displaying the radio buttons on the window.